go.mod

```diff
@@ -42,30 +43,29 @@ require (
 	github.com/paulmach/go.geojson v1.5.0
 	github.com/pkg/errors v0.9.1
 	github.com/pkg/profile v1.7.0
-	github.com/prometheus/client_golang v1.20.2
+	github.com/prometheus/client_golang v1.20.3
 	github.com/soheilhy/cmux v0.1.5
-	github.com/spf13/cast v1.3.1
+	github.com/spf13/cast v1.7.0
```

Comment:
please do not update these deps. There is a bug in cast library causing our tests to fail. More details here https://github.com/dgraph-io/dgraph/pull/9141